replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorHistoryImpl.java

```diff
@@ -36,8 +48,12 @@
 
```

Comment:
✏️ Class documentation and copyright notice

Reply:
Added class documentation